Week 1/Day 3/recipes/css/styles.css

```diff
@@ -0,0 +1,68 @@
+.apple-recipe-header{
+    background-image: url(../images/apple-pie.jpg);
+    background-size:cover ;
```

Comment:
Background-images. Background-size cover. You'll end up using this a million times.  Also see for future use `background-repeat: no-repeat;`